interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is usually before an argument.
 
-Researched answer:
+Researched answer: Arguments are data values that are passed when the method is called, while parameters is a variable that is defined when a function is declared.
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-Your answer:
+Your answer: .map() takes the original array and iterates through it giving out a new array that has updated values inside of array.
 
-Researched answer:
+Researched answer: map makes a new array with results of calling a funciton for each element inside of the old array, map calls given function once for every element in the array in order, map does not change the original array 
```

Comment:
Careful to not answer a different question, especially in the researched answer.